app/javascript/controllers/edit_deposit_controller.js

```diff
@@ -29,7 +30,7 @@ export default class extends Controller {
   displayErrors(event) {
     const [data, _status, _xhr] = event.detail;
     for (const [fieldName, errorList] of Object.entries(data)) {
-      const key = `${fieldName}Field`
+      const key = `${fieldName.replace("_", "-")}Field`
```

Comment:
Have you considered putting this transform logic go into `app/views/works/errors.json.jbuilder` instead?  We're already doing some field name transformation there.

Reply:
good call